src/gui/sdlmain.cpp

```diff
@@ -4190,6 +4185,13 @@ void Config_Add_SDL() {
 	        "             WxH format. For example: 1024x768.\n"
 	        "             Scaling is not performed for output=surface.");
 
+	pstring = sdl_sec->Add_path("viewport_resolution", always, "fill");
+	pstring->Set_help(
+	        "Set the viewport size (drawable area) within the window/screen:\n"
+	        "  fill:      Let the viewport fill the available window/screen (default).\n"
```

Comment:
Shouldn't that be "fit"? The setting isn't backwards compatible anyway.

Reply:
Fit is even better than fill; because for PP, we often don't fill.  Will drop that in.